CONTRIBUTING.md

```diff
@@ -48,6 +48,17 @@ load <shellScriptHere>.sh
 For more information on how to write bats tests, see the [bats
 documentation](https://bats-core.readthedocs.io/en/stable/writing-tests.html)
 as well as existing tests in this repository (search for `*_test.bats` files!)
+
+### Building and testing the CircleCI orb
+
+If you want to test changes to the shared orb:
+
+1. Make the changes to the orb
+2. Run `make publish-orb`
+3. In a test service, change the orb version (after the `@`,
```

Comment:
Note: If using `stencil-circleci` and `devbase` is set to `channel: main`, this part is done automatically by `stencil-circleci`.